package.json

```diff
@@ -9,14 +9,25 @@
     "Maarten van den Berg <>"
   ],
   "dependencies": {
+    "@fullhuman/postcss-purgecss": "^2.3.0",
+    "@rails/webpacker": "5.2.1",
     "bootstrap": "^4.3.1",
     "clipboard": "^1.6.1",
+    "core-js": "^3.6.5",
```

Comment:
I'm not actually sure what each of this packages do, so I'll have to clean this bit out